src/components/Channel/Channel.js

```diff
@@ -85,7 +88,7 @@ const Channel = (props) => {
 
     return () => {
       client.off('connection.recovered', handleEvent);
-      channel.off?.(handleEvent);
+      channel.off && channel.off(handleEvent);
```

Comment:
wouldn't this break on the line before so this isn't necessary here?
```suggestion
      channel.off(handleEvent);
```